components/playground/monitor.go

```diff
@@ -0,0 +1,125 @@
+package main
```

Comment:
The License header is missing. And I think it's better to move to `instance`.

Reply:
add by https://github.com/pingcap/tiup/pull/431/commits/b8901633a3b61746d70c003b3e8ceeb51ad9b218